pkg/nativeimgutil/fuzz_test.go

```diff
@@ -0,0 +1,33 @@
+package nativeimgutil
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func FuzzConvertToRaw(f *testing.F) {
+	f.Fuzz(func(t *testing.T, imgData []byte, withBacking, specifySize bool) {
+		if len(imgData) < 3 {
+			return
+		}
```

Comment:
Why is this condition here? Shouldn't the fuzzer be allowed test arbitrary data sizes?